src/views/CavaticaBixView.js

```diff
@@ -23,11 +23,15 @@ const CavaticaBixView = ({
   syncProjects,
   linkProject,
   unlinkProject,
+  history,
 }) => {
   const isAdmin = !user.loading
     ? user.myProfile.roles.includes('ADMIN')
     : false;
-  const [showModal, setShowModal] = useState('');
+  const hashOpenHook = (history, modalName) => {
```

Comment:
after we merge this let's create a PR to abstract this into a vanilla Modal wrapper component and migrate all modals to use it